### DISCUSSION TITLE
HASPone App Store

### DISCUSSION THREAD

**Discussion by aderusha** [HASwitchPlate/HASPone#6] [Show and tell]
[Check it out!](https://github.com/HASwitchPlate/HASPone/tree/main/Home_Assistant/blueprints)

**Comment by Cjdavidson** [comment]
Wow that was a lot easier to do. Thanks
If your Blueprints are updated will the ones i saved be updated too?

**Reply by aderusha** [reply]
Sadly, no.  It is on the Home Assistant roadmap and I've included the `source_url` key in each of the blueprints, so hopefully later we will be able to update automatically.
